.github/workflows/build.sh

```diff
@@ -2,8 +2,7 @@
 set -e -o pipefail
 export ZUO_JOBS="$(getconf _NPROCESSORS_ONLN)"
 if test "$TOOLCHAIN" = vs ; then
-    # cmd.exe /c "build.bat $TARGET_MACHINE"
-    echo assuming built previously
+    cmd.exe //c "build.bat $TARGET_MACHINE"
```

Comment:
Try `MSYS_NO_PATHCONV=1 cmd.exe /c build.bat $TARGET_MACHINE`

Reply:
Thanks — I see how that can be better. Change pushed.